shared/remediations/bash/templates/create_services_disabled.py

```diff
@@ -18,7 +18,7 @@
 
 def output_checkfile(serviceinfo):
     # get the items out of the list
-    servicename, packagename = serviceinfo
+    servicename, packagename,daemonname = serviceinfo
```

Comment:
IIRC, daemonname is only needed for OVAL templates, not remediations. Ref:

https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/oval/templates/create_services_disabled.py#L41